armory/scenarios/base.py

```diff
@@ -119,12 +120,11 @@ def _send_to_mongo(self, mongo_host: str, output: dict):
         """
         Send results to a Mongo database at mongo_host
         """
-        client = pymongo.MongoClient(mongo_host, MONGO_PORT)
+        client=pymongo.MongoClient(mongo_host,MONGO_PORT)
         db = client[MONGO_DATABASE]
         col = db[MONGO_COLLECTION]
-        logger.info(
-            f"Sending evaluation results to MongoDB instance {mongo_host}:{MONGO_PORT}"
-        )
+        mongo_ip=re.findall(r"@([^@]*$)", mongo_host)[0]
```

Comment:
Formatting looks off. Make sure that you have added the git commit pre hooks for black, flake8, etc.